components/Common/AvatarGroup/index.tsx

```diff
@@ -0,0 +1,41 @@
+'use client';
+import { useState } from 'react';
+import styles from './index.module.css';
+import type { FC } from 'react';
+import Avatar from './avatar';
+
+type AvatarGroupProps = {
+  avatars: string[];
+  limit?: number;
+  forceShow?: boolean;
+};
+
+const AvatarGroup: FC<AvatarGroupProps> = ({
+  limit = 10,
+  avatars,
+  forceShow,
+}) => {
+  const [showMore, setShowMore] = useState(forceShow?.valueOf() || false);
+
+  const handleShowMoreClick = () => {
+    if (forceShow) return;
```

Comment:
What is forceShow? What is is used for?

Reply:
to fort showing all avatars without taking attention to limit

Reply:
remove because not useful adding complexity for nothing